tests/tests/rap_rrtmgp_debug

```diff
@@ -0,0 +1,71 @@
+###############################################################################
+#
+#  RAP with RRTMGP debug test: atmosphere only at C96L127 in debug mode
+#
+###############################################################################
+
+export TEST_DESCR="Compare RAP RRTMGP debug results with previous trunk version"
+
+export CNTL_DIR=rap_rrtmgp_debug
+
+export LIST_FILES="sfcf000.nc \
+                   sfcf001.nc \
+                   atmf000.nc \
+                   atmf001.nc"
+
+export_fv3
+export NPZ=127
+export NPZP=128
+export DT_ATMOS=720
+export SYEAR=2021
+export SMONTH=03
+export SDAY=22
+export SHOUR=06
+export OUTPUT_GRID='gaussian_grid'
+export NSTF_NAME='2,0,0,0,0'
+export IAER=5111
+export FHMAX=1
+export OUTPUT_FH="0 1"
+#export WRITE_DOPOST=.true.
+
+export FV3_RUN=control_run.IN
+export CCPP_SUITE=FV3_RAP_RRTMGP
+export INPUT_NML=rap.nml.IN
+export FIELD_TABLE=field_table_thompson_aero_tke
+export NEW_DIAGTABLE=diag_table_rap
+
+export DT_ATMOS="150"
```

Comment:
here dt_atmos is set to 150?

Reply:
Yes. The 127 vertical levels really pose a problem for the long time steps in FV3 when using RAP physics plus RRTMGP.